aidial_sdk/chat_completion/request.py

```diff
@@ -31,23 +31,33 @@ class CustomContent(ExtraForbidModel):
     state: Optional[Any] = None
 
 
+class FunctionCall(ExtraForbidModel):
+    name: str
+    arguments: str
+
+
+class ToolCall(ExtraForbidModel):
+    # OpenAI API doesn't strictly specify existence of the index field
+    index: Optional[int]
+    id: StrictStr
+    type: Literal["function"]
+    function: FunctionCall
+
+
 class Role(Enum):
     SYSTEM = "system"
     USER = "user"
     ASSISTANT = "assistant"
     FUNCTION = "function"
-
-
-class FunctionCall(ExtraForbidModel):
-    name: str
-    arguments: str
+    TOOL = "tool"
 
 
 class Message(ExtraForbidModel):
     role: Role
     content: Optional[StrictStr] = None
-    custom_content: Optional[CustomContent] = None
```

Comment:
why is it removed? I think we still need this in the message model

Reply:
Fixed